python/spec/dependabot/python/file_updater/pipfile_file_updater_spec.rb

```diff
@@ -171,88 +169,14 @@
         let(:python_version_file) do
           Dependabot::DependencyFile.new(
             name: ".python-version",
-            content: "2.7.18\n"
+            content: "3.9.4\n"
           )
         end
 
         it "updates both files correctly" do
           expect(updated_files.map(&:name)).to eq(%w(Pipfile Pipfile.lock))
         end
       end
-
-      context "when the Python requirement is implicit" do
```

Comment:
This "implicit" python version means the update fails on py3 and we retry on py2, we don't do this anymore so test is yanked out